[BUG]make lint error

https://github.com/apecloud/kubeblocks/actions/runs/9872018658/job/27261231266
```
/home/runner/work/kubeblocks/kubeblocks/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./apis/...;./externalapis/..."
make golangci-lint
make[1]: Entering directory '/home/runner/work/kubeblocks/kubeblocks'
golangci-lint is already installed
/home/runner/work/kubeblocks/kubeblocks/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./apis/...;./externalapis/..."
/home/runner/go/bin/golangci-lint run ./...
apis/workloads/v1alpha1/instanceset_types.go:88: File is not `gofmt`-ed with `-s` (gofmt)
}
pkg/controller/instanceset/instance_util_test.go:31: File is not `gofmt`-ed with `-s` (gofmt)
	"k8s.io/apimachinery/pkg/util/sets"
make[1]: *** [Makefile:182: golangci-lint] Error 1
make[1]: Leaving directory '/home/runner/work/kubeblocks/kubeblocks'
make: *** [Makefile:178: lint] Error 2
Error: Process completed with exit code 2.
```

--- a/apis/apps/v1alpha1/scheduling_policy.go
+++ b/apis/apps/v1alpha1/scheduling_policy.go
@@ -1,3 +1,22 @@
+/*
+Copyright (C) 2022-2024 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
 package v1alpha1
 
 import corev1 "k8s.io/api/core/v1"

--- a/apis/workloads/v1alpha1/instanceset_types.go
+++ b/apis/workloads/v1alpha1/instanceset_types.go
@@ -86,6 +86,7 @@ type Ordinals struct {
 	Ranges   []Range `json:"ranges,omitempty"`
 	Discrete []int32 `json:"discrete,omitempty"`
 }
+
 // InstanceTemplate allows customization of individual replica configurations within a Component,
 // without altering the base component template defined in ClusterComponentSpec.
 // It enables the application of distinct settings to specific instances (replicas),

--- a/config/crd/bases/apps.kubeblocks.io_servicedescriptors.yaml
+++ b/config/crd/bases/apps.kubeblocks.io_servicedescriptors.yaml
@@ -273,7 +273,7 @@ spec:
               endpoint:
                 description: "Specifies the endpoint of the external service. \n If
                   the service is exposed via a cluster, the endpoint will be provided
-                  in the format of <host>:<port>."
+                  in the format of `host:port`."
                 properties:
                   value:
                     description: "Holds a direct string or an expression that can

--- a/deploy/helm/crds/apps.kubeblocks.io_servicedescriptors.yaml
+++ b/deploy/helm/crds/apps.kubeblocks.io_servicedescriptors.yaml
@@ -273,7 +273,7 @@ spec:
               endpoint:
                 description: "Specifies the endpoint of the external service. \n If
                   the service is exposed via a cluster, the endpoint will be provided
-                  in the format of <host>:<port>."
+                  in the format of `host:port`."
                 properties:
                   value:
                     description: "Holds a direct string or an expression that can

--- a/pkg/controller/instanceset/instance_template_util.go
+++ b/pkg/controller/instanceset/instance_template_util.go
@@ -1,3 +1,22 @@
+/*
+Copyright (C) 2022-2024 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
 package instanceset
 
 import (

--- a/pkg/controller/instanceset/instance_util_test.go
+++ b/pkg/controller/instanceset/instance_util_test.go
@@ -28,8 +28,8 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
-	"k8s.io/apimachinery/pkg/util/sets"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/utils/pointer"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
@@ -878,8 +878,8 @@ var _ = Describe("instance util test", func() {
 				},
 			}
 			affinity2 := &corev1.Affinity{
-					NodeAffinity: &corev1.NodeAffinity{
-						RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+				NodeAffinity: &corev1.NodeAffinity{
+					RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
 						NodeSelectorTerms: []corev1.NodeSelectorTerm{
 							{
 								MatchExpressions: []corev1.NodeSelectorRequirement{

